### PR TITLE
docs: update admin-api.md

### PR DESCRIPTION
**Pull request by Firstsawyou**
### What this PR does / why we need it:

Related issue  https://github.com/apache/apisix/issues/3642#issuecomment-784009522

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: doc/admin-api.md [hunk 1]
```diff
@@ -35,6 +35,8 @@
 
 *Description*：Route matches requests based on preset rules, and loads the appropriate plugin according to the matching result, then forwarding requests to target Upstream.
 
+Note: When `Admin API` is enabled, Apache APISIX will occupy the API prefixed with `/apisix/admin`. Therefore, when you design an API, please try to avoid using it.
```
Comment (juzhiyuan) [inline doc/admin-api.md:38]:
```suggestion
Note: When `Admin API` is enabled, Apache APISIX will occupy the API prefixed with `/apisix/admin`. Therefore, when you design an API, please AVOID using it.
```

Comment (juzhiyuan) [reply doc/admin-api.md:38]: if we use `try to` here, once users use this way, what should they do?

Comment (spacewander) [inline doc/admin-api.md:38]: Instead of avoiding `/apisix/admin`, we should warn user to choice a different port from the data panel.

**Review by spacewander — CHANGES_REQUESTED** [review]

File: doc/admin-api.md (hunk 1 above)
Comment (Firstsawyou) [reply doc/admin-api.md:38]: updated.  
The Admin API port can be customized through `port_admin` in `conf/config.yaml`.

Comment (Firstsawyou) [reply doc/admin-api.md]: updated.

File: doc/admin-api.md [hunk 2]
```diff
@@ -35,6 +35,8 @@
 
 *Description*：Route matches requests based on preset rules, and loads the appropriate plugin according to the matching result, then forwarding requests to target Upstream.
 
+Note: When the `Admin API` is enabled, it will occupy the API prefixed with `/apisix/admin`. Therefore, in order to avoid conflicts between your design API and `/apisix/admin`, it is recommended that the port of the data plane and the control plane be different. You can customize the Admin API port through `port_admin` in `conf/config.yaml`.
```
Comment (spacewander) [inline doc/admin-api.md:38]: it is recommended that the port of the data plane and the control plane be different.

=>

it is recommended to use a different port for the Admin API.

Comment (Firstsawyou) [reply doc/admin-api.md]: updated.

**Review by tokers — APPROVED** [review]

**Review by spacewander — APPROVED** [review]

**Pull request merged**